arch/risc-v/src/esp32c3/esp32c3_wifi_adapter.c

```diff
@@ -414,6 +410,11 @@ uint8_t esp_crc8(const uint8_t *p, uint32_t len);
 
 static bool g_wifi_irq_bind;
 
+/* Wi-Fi thread private data */
+
+static int g_wifi_thread_key;
+static bool g_wifi_tkey_init;
```

Comment:
Seems that we can make it private to `esp_thread_semphr_get`

Reply:
We could but this is how it was in esp32_wifi_adapter.c so I'll keep it like this. I don't want to touch this code too much.

Reply:
Sorry but there was no `g_wifi_thread_key`/`g_wifi_tkey_init` before this change. But we can keep it

Reply:
This way was copy&pasted from arch/xtensa/src/esp32/esp32_wifi_adapter.c